server/bleep/src/lib.rs

```diff
@@ -129,10 +158,14 @@ impl Application {
         Ok(Self {
             indexes: Arc::new(Indexes::new(config.clone(), semantic.clone())?),
             background: BackgroundExecutor::start(config.clone()),
+            prior_conversational_store: Arc::default(),
             repo_pool: config.source.initialize_pool()?,
             cookie_key: config.source.initialize_cookie_key()?,
             credentials: config.source.initialize_credentials()?.into(),
-            prior_conversational_store: Arc::default(),
+            user_store: config.source.load_or_default("user_store")?,
+            tracking_seed: config
```

Comment:
Just to be clear, how does this work with cloud deployments where there is more than 1 user per `Application`? What is the `tracking_seed` in this case?

Reply:
In cloud deployments, the `Application`-level tracking seed can be used to identify the instance. We don't expose this currently, but we can in the future.

If there are multiple users in the system, we will always use the per-user values in `user_store`.